archetypes/data-playground.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+subtitle: "add subtitle here" 
```

Comment:
```suggestion
subtitle: "Add Subtitle Here" 
```

Reply:
I believe that this change and the one below will make the text a little more consistent.